cdap-ui/app/cdap/components/FieldLevelLineage/v2/LineageSummary/index.tsx

```diff
@@ -250,10 +269,35 @@ class LineageSummary extends React.Component<{ classes }, ILineageState> {
               </div>
             );
           }
+
+          if (error) {
```

Comment:
we can do

if (error) return null;
return (
  <div>...</div>
);

Reply:
no we cannot. If there is error we want to display this, otherwise, there's already an actual component being rendered